### ISSUE TITLE
Extended package display support

### ISSUE THREAD

**Issue by iAdramelk** [smelukov/statoscope#15]
Hi. Thank you for the great lib, it allows me to have a lot more understanding about inner workings of dependency graph.

But right now all dependency graph analysis is based on per file basis, while I mostly use on per package basis. My most common usecases are:

1. Find why some library, e.g. `underscore` was included in the bundle (usually it was required by another package). Underscore can also be included not from main entry point, but by directly importing some method. Right now I need to test each included file to find what other libraries was using it.
2. Test if there two versions of some package in bundle e.g. lodash-2.* and lodash-5.*, and why.
3. Test if some dependencies exists in different chunks at the same time. For example React can be present in all chunks because it was required in each react component. And I will want to move it to common chunk. Also CommonsChunkPlugin by default will not move modules from children chunks to common chunk without special flag, e.g. So to be sure that there is no unwanted duplicates, I'd like some way to see duplicates list between chunks.
4. Understand how library added in one of my file affects total chunk size. For example: I added https://www.npmjs.com/package/normalize-object to bundle to convert objects from camelCase to snake_case, it required full `underscore` and couple other libraries, so for simple case conversion that can be done with 20 lines of code I extended bundle size by 30kb. To prevent such situations I'd like to be able to do following: 1) Have a list of external dependencies that I require from my own files (not all libraries in node_modules, only the ones I require myself). 2. Total size each dependency adds to bundle size (including imported libs). 3. List of places there I use each dependency.

So basically I'd like have a mode where I can see library from node_modules like it is a black box with total weight and with common list of requires from each of my files, and with list of library dependencies also in the form of black boxes, instead of separate files. Also some additional tools to for library analysis will be very useful.

Thanks in advance!

**Comment by smelukov** [comment]
Now we have a brand new analyzer in this repo, please check this out 😀
